src/patternfly/components/MenuToggle/examples/MenuToggle.md

```diff
@@ -249,10 +249,13 @@ import './MenuToggle.css'
 {{/menu-toggle}}
 ```
 
-### Split button (checkbox with label)
+### Split toggle with labeled checkbox
+
+To add a label to a split toggle that will be linked to the checkbox, pass the text wrapped in `span.pf-v6-c-check__label` as a child to `label.pf-v6-c-check`. Clicking the text will update the checked state of the split toggle's checkbox.
```

Comment:
Not sure since I can't load the surge preview right now to interact with this example, but the React version of this example doesn't update the checkbox when you click the text -- is this one different?

Reply:
Yeah, for this PR this example updates the checkbox when you click the text, while "Split toggle with checkbox and toggle text label" in React is the one that updates the checkbox when you click the text. So this PR is basically swapping the behaviors between the 2 examples ("Split toggle with checkbox and toggle text"/"Split toggle with checkbox and toggle text label" and "Split toggle with labeled checkbox")

Reply:
Ah okay, gotcha! I think? The naming of these in general is tough 😅

I think that we should try to match with v6 React, whether that means changing the React example names in a followup, or aligning in this pr. I personally think the language of "labeled checkbox" doesn't signify that clicking the text triggers selection as well as saying "toggle text" does. Or maybe even "label that toggles", "clickable text/label", "selectable text/label"?

But, I think I'm still confused. Your "Split toggle with checkbox and toggle text" example also says "Clicking the text should trigger any click action on the toggle." Is that new behavior, since the respective React "Split toggle with labeled checkbox" example doesn't trigger any action on clicks? Am I still mixing them up lol

Reply:
> The naming of these in general is tough

You are not wrong 😆 

Basically with this PR I was trying to convey that "Split toggle with labelled checkbox" ties the text label to the checkbox, and clicking the text will trigger the checkbox to be updated. Then "Split toggle with checkbox and toggle text" ties the text label to the toggle button. The "trigger click action on the toggle" is technically the default behavior of the split toggle, I don't think we actually show that in the example(s), though, since they're just stand alone Menu Toggles.

But yeah as long as we have Core and React matching in the end. It just depends what makes the most sense, which unfortunately is tough 😆

Reply:
Okay, _now_ I'm finally following -- ty for explaining so much 😁 In that case, I've changed my mind. I think the example titles that you use in this PR make the most sense! I can open up a followup pr to update the React naming to match